adhocracy4/comments/models.py

```diff
@@ -60,6 +63,8 @@ def __str__(self):
     def save(self, *args, **kwargs):
         """Change comment.comment if comment was marked removed or censored."""
         self.comment = transforms.clean_html_all(self.comment)
+        if not self.project:
```

Comment:
@Rineee do we need  to distinguish between comment and chapter here as well?

Reply:
I think this should work, because both comments and paragraphs have a module property that then goes to parent or chapter.